src/schema/comments.ts

```diff
@@ -37,6 +36,8 @@ import {
 import { ensureSourcePermissions, SourcePermissions } from './sources';
 import { generateShortId } from '../ids';
 import { CommentReport } from '../entity/CommentReport';
+import { UserComment } from '../entity/user/UserComment';
+import { UserVote } from '../types';
```

Comment:
Could have actually done `import type` i guess?

Reply:
We can't because it is used as value in the file.